### PR TITLE
Moved styles folder, refactored its classes and its call

### PR DESCRIPTION
**Pull request by lSindra**
Making changes requested in the issue: #984

Signed-off-by: Lucas Sindra <lucassindra01@hotmail.com>

### CODE REVIEW

File: packages/navigator/src/browser/navigator-frontend-module.ts [hunk 1]
```diff
@@ -14,6 +14,8 @@ import { FileNavigatorContribution } from './navigator-contribution';
 import { createFileNavigatorWidget } from "./navigator-container";
 import { WidgetFactory } from '@theia/core/lib/browser/widget-manager';
 
+import '/style/index.css';
```
Comment (kittaakos) [inline packages/navigator/src/browser/navigator-frontend-module.ts]: The generated `.js` file will be put under `navigator/lib/browser/navigator-frontend-module.js` while the CSS file remains under `navigator/workspace/src/browser/style/index.css`. Hence you need to traverse up from the `lib` folder to the Theia extension root, then locate the CSS file with the `import '../../src/browser/style/index.css';` statement.

I believe, the incorrect CSS location causes the build failures currently.  See for example [here](https://travis-ci.org/theia-ide/theia/jobs/316740096#L1117). If you have questions, feel free to ask, I am happy to help you with the onboarding.

**Comment by kittaakos** [comment]
@lSindra, thank you for your contribution. You are almost there. I have added a short comment. After fixing the CSS path in the navigator module, please squash your commits. So we end up with one single change once we merge your fixes to the master branch.

File: packages/navigator/src/browser/navigator-frontend-module.ts (hunk 1 above)
Comment (lSindra) [reply packages/navigator/src/browser/navigator-frontend-module.ts]: Oh I see, thanks for the input. Will be making these changes asap.

**Comment by lSindra** [comment]
Maybe i've missed something? My local repo is working fine, but the tests are not.

**Comment by kittaakos** [comment]
The build failures seem to be unrelated to your changes. I have restarted both CIs, let's wait for another test result.

**Comment by akosyakov** [comment]
LGTM, thank you

**Pull request merged**